workflow-templates/im-build-and-test-generic.yml

```diff
@@ -0,0 +1,328 @@
+# Workflow Code: MadLark_v1    DO NOT REMOVE
+# Purpose:
+#    Provide a framework for building and testing multiple project types in 
+#    this repository in parallel jobs in response to pull requests.
+#    Each build-test job should check out the code, build one project, run its
+#    associated tests and upload the packaged binaries as an artifact.
+#    After all the build-test jobs have completed, the create-release job
+#    will upload the built artifacts to a GitHub release asset.
```

Comment:
Should this template only be implemented by the start-project process? I guess a team could manually implement it, but then some of the `(don't delete this comment)` comments won't make be applicable (if I'm understanding things correctly). Do we want to give further instruction along those lines?

Reply:
Because this template is specifically built to work with `start-project`, and `start-project` can add it to all the repos it touches, I think I'm going to actually remove it from this repo and add it to start-project itself instead.